packages/core/src/progress-circle/register.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { registerElementSafely } from '@clr/core/internal';
+import { CdsProgressCircle } from '@clr/core/progress-circle';
```

Comment:
nit: local entrypoint files should be relative

```
import { CdsProgressCircle } from './progress-circle.element.js';
```

Reply:
✅